Classes/Domain/Repository/LegacyCountryRepository.php

```diff
@@ -0,0 +1,89 @@
+<?php
+namespace PAGEmachine\Ats\Domain\Repository;
+
+use PAGEmachine\Ats\Service\IntlLocalizationService;
+
+/*
+ * This file is part of the PAGEmachine ATS project.
+ */
+
+/**
+ * Repository for country data (static info tables) - Legacy version for TYPO3 7 without doctrine
+ */
+class LegacyCountryRepository
+{
+    public function findAll()
+    {
+        $countries = $GLOBALS['TYPO3_DB']->exec_SELECTgetRows(
+            implode(',', ['uid', 'cn_iso_2', 'cn_iso_3', 'cn_short_en', 'cn_short_local']),
+            'static_countries',
+            'deleted = 0'
+        );
+
+        $localizationService = IntlLocalizationService::getInstance();
+
+        foreach ($countries as $key => $country) {
+            $countries[$key]['localizedName'] = $localizationService->getLocalizedRegionName($country['cn_iso_2']) ?: $country['cn_short_local'];
+        }
+
+        return $countries;
+    }
+
+    /**
+     * Finds languages by their respective uids
+     *
+     * @param array $uids
+     * @return array $countries
+     */
+    public function findCountriesByUids($uids = [])
+    {
+        $countries = $GLOBALS['TYPO3_DB']->exec_SELECTgetRows(
+            implode(',', ['uid', 'cn_iso_2', 'cn_iso_3', 'cn_short_en', 'cn_short_local']),
+            'static_countries',
+            'deleted = 0 AND uid IN(' . implode(',', $uids) . ')'
```

Comment:
There must be a way to at least escape these. If possible use a `PreparedStatement` instead.

Reply:
Escaped them to int (by uid) and regex A-Z (by country code).